engine/common/requester/engine.go

```diff
@@ -319,7 +317,12 @@ func (e *Engine) dispatchRequest() (bool, error) {
 	for k := range e.items {
 		rndItems = append(rndItems, e.items[k].EntityID)
 	}
-	e.rng.Shuffle(len(rndItems), func(i, j int) { rndItems[i], rndItems[j] = rndItems[j], rndItems[i] })
+	err = rand.Shuffle(uint(len(rndItems)), func(i, j uint) {
```

Comment:
Please add error documentation to `utils/rand` and handling here.

Reply:
[a89aeab](https://github.com/onflow/flow-go/pull/4258/commits/a89aeabdfe72e47abb7a381f9ff91b3414e44989)